plugins/services/src/js/reducers/__tests__/JSONMultiContainer-test.js

```diff
@@ -0,0 +1,84 @@
+const Batch = require("#SRC/js/structs/Batch");
+const { combineParsers } = require("#SRC/js/utils/ParserUtil");
+const { combineReducers } = require("#SRC/js/utils/ReducerUtil");
+const JSONMultiContainerParser = require("../JSONMultiContainerParser");
+const JSONMultiContainerReducers = require("../JSONMultiContainerReducers");
+
+describe("JSONMultiContainer", function() {
```

Comment:
I was thinking that it might make sense to check the parse to reducer (JSON>Batch>JSON) flow in one file and the reducer to parser (Batch>JSON>Batch) in another. But I'm fine with whatever you favor.